assets/src/components/site-scan-results/site-scan-sources-detail.js

```diff
@@ -27,7 +27,7 @@ import ClipboardButton from '../clipboard-button';
  */
 function getAssociatedSources(sources, slug) {
 	return sources.filter((source) => {
-		return slug === getPluginSlugFromFile(source.name);
+		return slug === getPluginSlugFromFile(source?.name);
```

Comment:
Wouldn't this make more sense?

```suggestion
		return source && slug === getPluginSlugFromFile(source.name);
```

Reply:
As if `source` is `null`, then `source?.name` would return `undefined`, and there's no point of passing `undefined` to `getPluginSlugFromFile()` since it will always be false. And actually, it would be an error since the function expects a string:

https://github.com/ampproject/amp-wp/blob/0e6dcfbe65056752c72af0a3c3a98769b02b8b23/assets/src/common/helpers/get-plugin-slug-from-file.js#L14-L16

Reply:
Or then perhaps better:

```suggestion
		return source?.name && slug === getPluginSlugFromFile(source.name);
```

Reply:
Yes, my solution can break things. I have fixed it with yours.